site/src/components/timetable/index.tsx

```diff
@@ -67,11 +70,13 @@ export function Timetable({ trains, ...props }: TimetableProps) {
               animation={{
                 delay: difference / DELAY_DIVIDEND
               }}
+              stations={stations}
               cancelledText={t('cancelled')}
               lastStationId={props.lastStationId ?? ''}
+              stationShortCode={props.stationShortCode}
               locale={locale}
               train={train}
-              key={`${train.trainNumber}-${train.departureDate}.${train.scheduledTime}`}
+              key={`${train.departureDate}-${train.trainNumber}-${props.stationShortCode}`}
```

Comment:
Key is not unique enough. A train can depart twice in the same day and the key would fail in this case.